include/wx/generic/grid.h

```diff
@@ -1481,6 +1481,22 @@ struct WXDLLIMPEXP_CORE wxGridSizesInfo
     wxUnsignedToIntHashMap m_customSizes;
 };
 
+// ----------------------------------------------------------------------------
+// wxGridArea
+// ----------------------------------------------------------------------------
+enum wxGridArea
+{
+    Corner      = 0x001, // m_cornerLabelWin
```

Comment:
The elements of the enum should have `wxGridArea_` prefix as otherwise they will inevitably conflict with something with the names as common as that. This is especially important if they remain in the public header, but they should have a prefix even if they're private because an identifier like `All` just can't be used in the global scope (but if they were private we could put the entire enum into a namespace perhaps).

Reply:
I'll use `wxGA_` as prefix then for shorter names unless you prefer the suggested one ?